content/scout/guides/vex.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Suppress image vulnerabilities with VEX
+description: An introduction to using VEX with Docker Scout
+keywords: scout, vex, Vulnerability Exploitability eXchange, openvex, vulnerabilities, cves
+---
+
+In this guide, you will learn about:
+
+- How VEX can help you suppress non-applicable or fixed vulnerabilities found in your images
+- How to create VEX documents and statements
+- How to apply and consume VEX data with Docker Scout
+
+## Prerequisites
+
+If you want to follow along the steps of this guide, you'll need the following:
+
+- Docker Desktop version 4.27.2 or later
```

Comment:
Can we add a link to Docker Desktop release notes page?

Reply:
Rephrasing this to "latest version" since that will always be applicable.